### ISSUE TITLE
[MU4 Issue] Dropdown lists should be treated as a UI subsection

### ISSUE THREAD

**Issue by shoogle** [musescore/MuseScore#8136]
Navigation away from a dropdown list should require use of the Tab, F6, Left or Right arrow keys. The Up and Down arrow keys should be reserved for interaction with dropdown itself.

## Steps

1. Open the New Score dialog.

2. Use Tab/Shift+Tab to navigate to the Genres dropdown list (under "Family").

![image](https://user-images.githubusercontent.com/11011881/166105875-2a13d6ae-b9d2-4b87-ab01-2a6c738e476a.png)

3. Press the Down arrow key.

## Expected behaviour

There are two options here based on behaviour of other applications (see **Additional Context** for examples).

1. The dropdown could open with the currently selected item highlighted (equivalent to pressing Spacebar).
  ![image](https://user-images.githubusercontent.com/11011881/166106337-2a16ed86-69e3-4b44-abe6-a5a455dc2b8b.png)

2. The dropdown could remain closed, but cycle to the next value (equivalent to Spacebar, Down arrow, Spacebar).
  ![image](https://user-images.githubusercontent.com/11011881/166106387-1503f5db-27f8-4538-aea9-e88b2b51556f.png)

My vote would be for option 2 because Spacebar is available if users want to see the contents of the list.

## Actual behaviour

The focus moves away from the Genres dropdown and into the Groups list below.

![image](https://user-images.githubusercontent.com/11011881/166106048-d022e1f4-58d4-4e52-aaf9-1784eaabc96a.png)

**Desktop (please complete the following information):**
 - OS: Windows and macOS

**Additional Context**

Examples of each behaviour in other applications:

1. Dropdown **opens** on Down arrow:
    - WordPad: Font and Font Size dropdowns (press Alt key to focus the ribbon then Tab to the dropdowns)
    - Discord: User Settings > Voice & Video (user settings is cog wheel near to profile image in bottom left of screen)
2. Dropdown **cycles** on Down arrow:
    - MuseScore 3 (and other Qt Widgets applications): Genres dropdown in NSW, Language dropdown in Preferences
    - Firefox: Settings > Home (`about:preferences#home`)
    - Edge: Settings > Default Browser (`edge://settings/defaultBrowser`)

**Comment by HemantAntony** [comment]
Here's a workaround (or maybe the way it was intended?):
1. Move to the genre dropdown.
2. Press the space key.
3. Then use the down arrow.

**Comment by Tantacrul** [comment]
@shoogle - are you sure about this or has your thinking changed?

I can confirm the behaviour you are describing is present in other apps that have dropdowns. I do notice that (as @HemantAntony points out) you can press 'Spacebar' to open them and esc. to escape them.

If you are happy with the way it is now, lemme know and I'll close this. If now, we can assign a priority and target it at Beta.

**Comment by shoogle** [comment]
@Tantacrul, this could actually work one of 2 ways. I've updated the description to include both methods and examples of each behaviour in other applications.

However, I consider this low priority given that Spacebar is available to interact with the dropdown, so I'm happy to leave it as it is and see if users complain. Feel free to close it if you like. I've added the "improvement" tag so we can find it later.
